students/johnpharmd/lesson06/mailroomtest.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+import unittest
+import mailroom4
+import os
+import datetime
+
+
+class MailroomTest(unittest.TestCase):
+    def make_test_list(self):
+        self.test_list = mailroom4.get_donor_list()
+        return self.test_list
+
+    def test_get_donor_list(self):
+        self.make_test_list()
+        assert type(self.test_list) == list
+
+    def test_donor_list_strings(self):
+        self.make_test_list()
+        for s in self.test_list:
+            if '-' not in s:
+                assert s.isalpha()
+            else:
+                assert s.replace('-', '').isalpha()
```

Comment:
if you wanted to test `get_donor_list` function here is an example of a test:
```
def test_get_donor_list():
    actual = mailroom4.get_donor_list()
    expected = ['Gates', 'Brin', 'Cerf', 'Musk', 'Berners-Lee', 'Wojcicki', 'Avey']
    self.assertEquals(actual, expected)
```

in very basic terms you are typically testing actual with expected

`make_test_list` isn't all that helpful and if you wanted to make something available to all tests then use `setUp` method.